app/models/about.rb

```diff
@@ -66,4 +80,20 @@ def stats
     }
   end
 
+  def category_moderators
+    category_ids = Guardian.new(@user).allowed_category_ids
+    return [] if category_ids.blank?
+    results = DB.query(<<~SQL, category_ids: category_ids)
+      SELECT c.id category_id, array_agg(gu.user_id) user_ids
+      FROM categories c
+      JOIN group_users gu
+      ON gu.group_id = reviewable_by_group_id
+      WHERE c.id IN (:category_ids)
+      GROUP BY c.id
+    SQL
+    moderators = User.where(id: results.map(&:user_ids).flatten).map { |u| [u.id, u] }.to_h
```

Comment:
```suggestion
    moderators = {}
    User.where(id: results.map(&:user_ids).flatten).each do |user|
      moderators[user.id] = user
    end
    moderators
```

This reduces multiple array allocations